README.md

```diff
@@ -1,77 +1,22 @@
 ![Testkube Logo](https://raw.githubusercontent.com/kubeshop/testkube/main/assets/testkube-color-gray.png)
 
-# Welcome to TestKube Template Executor
-
-TestKube Template Executor is a test executor skeleton for [TestKube](https://testkube.io).  
-You can use it as basic building blocks for creating a new executor.
-
-# What is an Executor?
-
-Executor is nothing more than a program wrapped into Docker container which gets JSON (testube.Execution) OpenAPI based document as an input and returns a stream of JSON output lines (testkube.ExecutorOutput), where each output line is simply wrapped in this JSON, similar to the structured logging idea. 
+# Welcome to TestKube Playwright Executor
```

Comment:
Could you please update the casing of `TestKube` to `Testkube`? I see it's wrong in our templates as well, I will take care of that one :)